conf/igenomes.config

```diff
@@ -295,6 +295,7 @@ params {
             bowtie2     = "${params.igenomes_base}/Homo_sapiens/UCSC/hg19/Sequence/Bowtie2Index/"
             star        = "${params.igenomes_base}/Homo_sapiens/UCSC/hg19/Sequence/STARIndex/"
             bismark     = "${params.igenomes_base}/Homo_sapiens/UCSC/hg19/Sequence/BismarkIndex/"
+            hisat2      = "s3://genome-idx/hisat/hg19_genome.tar.gz"
```

Comment:
Is there a reason this one must use a separate URL rather than the base?

Reply:
Yeah, it does because it's not hosted under igenomes. 

https://daehwankimlab.github.io/hisat2/download/

We could move it under igenomes, but I'm trying to avoid shaving that yak until we work on nf-core/references a bit more.